composer.json

```
@@ -50,7 +50,7 @@
         "monolog/monolog": "^2.9||^3.0",
         "psr/http-message": "^1.0|^2.0",
         "ramsey/uuid": "^4.0",
-        "google/gax": "^1.30",
+        "google/gax": "dev-main as 1.32",
```

This will need to be `^1.32` once we merge https://github.com/googleapis/gax-php/pull/562 
```suggestion
        "google/gax": "^1.32",
```